siliconcompiler/sphinx_ext/dynamicgen.py

```diff
@@ -405,18 +405,20 @@ def display_config(self, chip, modname):
             section_key = '-'.join(['flows', modname, step])
             section = build_section(step, section_key)
             step_cfg = {}
-            for prefix in ['flowgraph']:
-                cfg = chip.getdict(prefix, step)
-                if cfg is None:
-                    continue
-                schema = Schema(cfg=cfg)
-                schema.prune()
-                pruned = schema.cfg
-                if prefix not in step_cfg:
-                    step_cfg[prefix] = {}
-                step_cfg[prefix][step] = pruned
-
-            section += build_schema_value_table(step_cfg, self.env.docname, skip_zero_weight=True)
+            cfg = chip.getdict('flowgraph', chip.design, step)
```

Comment:
Just building the main flowgraph is enough here?
Not sure what the original intention was with building the additional flowgraphs as well...